bigtable/metrics.go

```diff
@@ -109,6 +111,12 @@ var (
 			},
 			recordedPerAttempt: true,
 		},
+		metricNameDebugTags: {
+			additionalAttrs: []string{
+				metricLabelKeyTag,
+			},
+			recordedPerAttempt: true,
```

Comment:
Where is this metric updated? I don't think we want to record a value per attempt, we only want to increment the value when there's an error with the async refresh.

Reply:
That field was not getting used for `debug_tags` metric. I have removed it now.

**More detail**:
What internally happens in the Go client library when user creates a client and invokes `ReadRows` method :
1. Create client
    1.1. Create tracerFactory
    1.2. Create attributes to be added to the `debug_tags` metric
    1.3. Create callback function to run when async refresh fails. This callback function records a change to the counter with the attributes created in 1.2
    1.4. Pass the callback function in the `EnableAsyncRefreshDryRun` client option while creating the GRPC dial pool
2. ReadRows
    2.1. Create metrics tracer
    2.2. Obtain Token. Run the callback function if any errors (not done by Bigtable client but by the auth package)
    2.3. Make ReadRows RPC call
    2.4. Receive Response
    2.5. After ReadRows operation completes, invoke `func (mt *builtinMetricsTracer) toOtelMetricAttrs(metricName string) ([]attribute.KeyValue, error)` 4 times, once with each metric name: operation_latencies, attempt_latencies, server_latencies and retry_count
    2.6. `toOtelMetricAttrs` uses the data saved in metrics tracer to create attributes for the metrics. If the metric needs to be recorded per attempt, include attempt status, else include operation status

As can be seen from above flow, ` toOtelMetricAttrs` is never called with `debug_tags` metric name. So, `recordedPerAttempt` was not getting used. Since, it was also incorrectly set to true, I have removed it now.

Also wanted to highlight that since the `debug_tags` metric is getting recorded long before the response is received, its table, cluster and zone attributes will be <unspecified>